### PR TITLE
Audit Logs Page Implementation

### PR DESCRIPTION
**Pull request by Renuka9527**
1. Added new Audit log page under the logs section of the side bar menu.
2. Jira Story: https://jsw.ibm.com/browse/PFEBMC-1099
Screenshot:
image
image

### CODE REVIEW

**Review by SteffiAntony — APPROVED** [review]

File: src/locales/en-US.json [hunk 1]
```diff
@@ -204,6 +206,24 @@
...
+      "addr": "Address",
+      "res": "Response",
+      "messageArgs": "MessageArgs",
+      "message": "Message",
+      "searchLogs": "Search logs",
+      "eventTimeStamp": "Date"
+    },
+    "toast": {
+      "infoStartDownload": "Download might take several minutes to complete.",
+      "errorStartDownload": "Error downloading the Additional data.",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: I think we have to rephrase this: "Error downloading Logs"

File: src/locales/en-US.json [hunk 2]
```diff
@@ -204,6 +206,24 @@
...
+      "successStartDownload":  "Additional data downloaded successfully."
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: I think we have to rephrase this: "Logs downloaded successfully"

File: src/locales/en-US.json (hunk 1 above)
Comment (Renuka9527) [reply src/locales/en-US.json]: Addressed

File: src/locales/en-US.json (hunk 2 above)
Comment (Renuka9527) [reply src/locales/en-US.json]: Addressed

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**